# inputs
int64 commond
---
# outputs
bool saved
